packages/bsky/src/api/app/bsky/feed/getTimeline.ts

```diff
@@ -102,9 +106,14 @@ const presentation = (inputs: {
   hydration: HydrationState
 }) => {
   const { ctx, skeleton, hydration } = inputs
-  const feed = mapDefined(skeleton.items, (item) =>
-    ctx.views.feedViewPost(item, hydration),
-  )
+  const feed = mapDefined(skeleton.items, (item) => {
+    const post = ctx.views.feedViewPost(item, hydration)
+    if (!post) return
+    if (isFeedPostHiddenReply({ post, hydration })) {
```

Comment:
i think this would be comfier in the "rules" part of the pipeline

(arguably the rules & presentation parts could be collapsed into a single step, but since they're separate for now I think we move it there)

Reply:
Moved to View